### PR TITLE
HPCC-14292 Auto refresh button needs tooltip

### PR DESCRIPTION
**Pull request by miguelvazq**
Add a tooltip to auto refresh and whether it is on/off.

Signed-off by: Miguel Vazquez miguel.vazquez@lexisnexis.com

### CODE REVIEW

**Comment by miguelvazq** [comment]
@GordonSmith Please review.

**Comment by miguelvazq** [comment]
@GordonSmith Can you verify?

File: esp/src/eclwatch/ActivityWidget.js [hunk 1]
```diff
@@ -583,6 +586,15 @@ define([
...
             this.wuMoveDownButton.set("disabled", !wuCanDown);
             this.wuMoveBottomButton.set("disabled", !wuCanDown);
+        },
+
+        createStackControllerTooltip: function (widgetID, text) {
+            return new Tooltip({
+                connectId: [widgetID],
+                label: text,
+                showDelay: 1,
+                position: ["below"]
```
Comment (GordonSmith) [inline esp/src/eclwatch/ActivityWidget.js]: Have we used tooltip in other places?  If so did we position it above or below?

**Comment by GordonSmith** [comment]
Looks fine - just one question on the positioning of the tooltip.

File: esp/src/eclwatch/ActivityWidget.js (hunk 1 above)
Comment (miguelvazq) [inline esp/src/eclwatch/ActivityWidget.js]: I am using tooltip position similar to the main icons (bottom). We do however use a "title" type attribute in other tooltips such as the ones on tabs but, they do not have the same appearance.

**Pull request merged**